transmission/updateSettings.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+import json
+
+
+# Verify script arguments
```

Comment:
`argparse` does this quite nicely. Gives you type checking, nice error messages and nicely formatted `help`